e2e/credentials-server.e2e.spec.ts

```diff
@@ -25,11 +25,27 @@ import path from 'path';
 import { expect, test } from '@playwright/test';
 import { BrowserContext, ElectronApplication, Page, _electron } from 'playwright';
 
-import fetch, { RequestInit } from 'node-fetch';
+import fetch from 'node-fetch';
 import { createDefaultSettings, playwrightReportAssets } from './utils/TestUtils';
 import paths from '@/utils/paths';
 import { ServerState } from '@/main/commandServer/httpCommandServer';
 import { spawnFile } from '@/utils/childProcess';
+import { findHomeDir } from '@/config/findHomeDir';
+
+function haveDockerCredentialAssistant(): boolean {
+  const dockerConfigPath = path.join(findHomeDir() ?? '', '.docker', 'config.json');
+
+  try {
+    const contents = JSON.parse(fs.readFileSync(dockerConfigPath).toString());
+    const credStore = contents.credsStore;
+
+    return !!credStore;
```

Comment:
Don't you want to test that the helper executable exist on the `PATH`?

E.g. the user may have `credsStore` configure to `desktop`, but have uninstalled DD, so won't have `docker-credential-desktop` available anymore.

Reply:
I added that code -- try to run it, and skip tests if it's not runnable.